backend/store/principal.go

```diff
@@ -417,11 +425,13 @@ func (s *Store) UpdateUser(ctx context.Context, userID int, patch *UpdateUserMes
 	defer tx.Rollback()
 
 	user := &UserMessage{}
+	var idpResourceID sql.NullString
+	var idpUserInfo string
 	if err := tx.QueryRowContext(ctx, fmt.Sprintf(`
 			UPDATE principal
 			SET `+strings.Join(principalSet, ", ")+`
 			WHERE id = $%d
-			RETURNING id, email, name, type, password_hash
+			RETURNING id, email, name, type, password_hash, idp_id, idp_user_info
```

Comment:
Isn't `idp_id` column has type `integer`? but we try to scan as a string? 🤔

Reply:
fixed